docs/src/pages/getting-started/installation/installation.md

```diff
@@ -67,7 +67,7 @@ affecting performance and bandwidth utilisation.
 
 We are providing two Universal Module Definition (UMD) files:
 
-- one for development: https://unpkg.com/@material-ui/core@next/umd/material-ui.development.js
-- one for production: https://unpkg.com/@material-ui/core@next/umd/material-ui.production.min.js
+- one for development: https://unpkg.com/@material-ui/core/umd/material-ui.development.js
```

Comment:
I think that we should keep the tag: latest now. People relying on the CDN version in production should use a fixed version. I think that it can help them.